source/LineShader.cpp

```diff
@@ -132,3 +132,16 @@ void LineShader::Draw(const Point &from, const Point &to, float width, const Col
 	glBindVertexArray(0);
 	glUseProgram(0);
 }
+
+
+
+void LineShader::DrawDashed(const Point &from, const Point &to, const Point &unit, const float width,
```

Comment:
I am fine with this, but you should add the gap width and the dash length as arguments imo

Reply:
I see your point (generalized routines being good and all), but perhaps we should leave that until someone needs it?

Reply:
I dont really agree on that point, but maybe you should wait on other peoples opinions